constrained_linear_regression/__init__.py

```diff
@@ -78,6 +84,195 @@ def fit(self, X, y, min_coef=None, max_coef=None):
         return self
 
 
+class MultiConstrainedLinearRegression(ConstrainedLinearRegression):
```

Comment:
With this new class, `__init__.py` becomes too big to be really readable. 
I suggest, you do the following refactoring:
* Move `ConstrainedLinearRegression` into a new file `constrained_linear_regression.py`
* Move `MultiConstrainedLinearRegression` into a new file `multi_constrained_linear_regression.py` that imports `constrained_linear_regression`
* Make `__init__.py` import both these classes.